### PR TITLE
Refactoring for compliance with the new API, store, service, controller and manager patterns.

### PR DESCRIPTION
**Pull request by tomikazi**
I've removed the naive controller implementation for now as it added no value in its initial state.
The bulk YAML load CLI was re-worked to utilize one Create at a time since the top-level topo API has changed and does not support composite operations at the moment. That is something that can be added later.
The get YAML command - which was already marked as deprecated - was removed entirely.
The device API and store were left as is since they are deprecated.

### CODE REVIEW

File: cmd/onos-topo/onos-topo.go [hunk 1]
```diff
@@ -49,50 +28,25 @@ func main() {
...
-}
 
-// Creates gRPC server and registers various services; then serves.
-func startServer(caPath string, keyPath string, certPath string) error {
-	oidcServer := os.Getenv(auth.OIDCServerURL)
-	securityConfig := northbound.SecurityConfig{}
-	if oidcServer != "" {
-		log.Infof("Authentication checking enabled. Using oidc server %s", oidcServer)
-		securityConfig.AuthenticationEnabled = true
+	log.Info("Starting onos-e2sub")
```
Comment (SeanCondon) [inline cmd/onos-topo/onos-topo.go:39]: maybe it should say 'topo'

File: cmd/onos-topo/onos-topo.go [hunk 2]
```diff
@@ -49,50 +28,25 @@ func main() {
...
-		return err
-	}
-	s.AddService(deviceService)
-
-	topoService, err := topo.NewService()
-	if err != nil {
-		return err
-	}
-	s.AddService(topoService)
+	log.Info("Starting onos-e2sub")
```
Comment (SeanCondon) [inline cmd/onos-topo/onos-topo.go:47]: and again

**Review by SeanCondon — APPROVED** [review]
minor comments

File: cmd/onos-topo/onos-topo.go (hunk 1 above)
Comment (tomikazi) [reply cmd/onos-topo/onos-topo.go:39]: Yes, it should.

File: pkg/cli/crud.go [hunk 3]
```diff
@@ -77,6 +77,8 @@ func getLoadCommand() *cobra.Command {
 		Use:   "load {topofile}",
 		Short: "Bulk load topo data from a file",
 	}
+
+	cmd.AddCommand(getLoadYamlCommand())
```
Comment (adibrastegarnia) [inline pkg/cli/crud.go:81]: You can remove it actually.

**Review by adibrastegarnia — APPROVED** [review]

**Pull request merged**